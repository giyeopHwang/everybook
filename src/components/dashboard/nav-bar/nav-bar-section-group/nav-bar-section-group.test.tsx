import { renderWithWrapper } from '@/test/test-utils';
import NavBarSectionGroup from './nav-bar-section-group.component';

describe('NavBarSectionGroup', () => {
  it('renders NavBarSectionGroup component', () => {
    const navBarSectionGroup = renderWithWrapper(<NavBarSectionGroup />);
    expect(navBarSectionGroup).toMatchSnapshot();
  });
});
