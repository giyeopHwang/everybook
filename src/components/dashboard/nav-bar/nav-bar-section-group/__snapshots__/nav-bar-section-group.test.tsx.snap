// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NavBarSectionGroup > renders NavBarSectionGroup component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="css-14chwgd"
      >
        <header
          class="css-kympkc"
        >
          <div
            class="css-359auk"
          >
            <svg
              aria-hidden="true"
              data-slot="icon"
              fill="none"
              stroke="currentColor"
              stroke-width="1.5"
              style="width: 1rem; height: 1rem;"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="m4.5 15.75 7.5-7.5 7.5 7.5"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
            <h2
              class="css-1bt39ed"
            >
              이달의 자산현황
            </h2>
          </div>
        </header>
        <div
          class="css-7ttcrf"
        >
          <div
            class="css-135bpbl"
          >
            <header
              class="css-kvjl0w"
            >
              <div
                class="css-vmk80s"
              >
                <svg
                  aria-hidden="true"
                  data-slot="icon"
                  fill="none"
                  stroke="currentColor"
                  stroke-width="1.5"
                  style="width: 0.9rem; height: 0.9rem;"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M3.75 9h16.5m-16.5 6.75h16.5"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                </svg>
                <h4
                  class="css-pqtvky"
                >
                  순자산
                </h4>
              </div>
              <span>
                500,000,000
              </span>
            </header>
            <section
              class="css-1r2jysi"
            />
          </div>
          <div
            class="css-135bpbl"
          >
            <header
              class="css-1r6e9a4"
            >
              <div
                class="css-vmk80s"
              >
                <svg
                  aria-hidden="true"
                  data-slot="icon"
                  fill="none"
                  stroke="currentColor"
                  stroke-width="1.5"
                  style="width: 0.9rem; height: 0.9rem;"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M12 4.5v15m7.5-7.5h-15"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                </svg>
                <h4
                  class="css-pqtvky"
                >
                  자산
                </h4>
              </div>
              <span>
                500,000,000
              </span>
            </header>
            <section
              class="css-1r2jysi"
            >
              <div
                class="css-uf1ume"
              >
                <span>
                  └ 
                  현금잔액
                </span>
                <span>
                  100,000,000
                </span>
              </div>
              <div
                class="css-uf1ume"
              >
                <span>
                  └ 
                  투자자산
                </span>
                <span>
                  350,000,000
                </span>
              </div>
              <div
                class="css-uf1ume"
              >
                <span>
                  └ 
                  기타자산
                </span>
                <span>
                  50,000,000
                </span>
              </div>
            </section>
          </div>
          <div
            class="css-135bpbl"
          >
            <header
              class="css-1i1w77g"
            >
              <div
                class="css-vmk80s"
              >
                <svg
                  aria-hidden="true"
                  data-slot="icon"
                  fill="none"
                  stroke="currentColor"
                  stroke-width="1.5"
                  style="width: 0.9rem; height: 0.9rem;"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M5 12h14"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                </svg>
                <h4
                  class="css-pqtvky"
                >
                  부채
                </h4>
              </div>
              <span>
                100,000,000
              </span>
            </header>
            <section
              class="css-1r2jysi"
            >
              <div
                class="css-uf1ume"
              >
                <span>
                  └ 
                  대출
                </span>
                <span>
                  75,000,000
                </span>
              </div>
              <div
                class="css-uf1ume"
              >
                <span>
                  └ 
                  기타부채
                </span>
                <span>
                  25,000,000
                </span>
              </div>
            </section>
          </div>
        </div>
      </section>
      <section
        class="css-14chwgd"
      >
        <header
          class="css-kympkc"
        >
          <div
            class="css-359auk"
          >
            <svg
              aria-hidden="true"
              data-slot="icon"
              fill="none"
              stroke="currentColor"
              stroke-width="1.5"
              style="width: 1rem; height: 1rem;"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="m4.5 15.75 7.5-7.5 7.5 7.5"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
            <h2
              class="css-1bt39ed"
            >
              이달의 가계
            </h2>
          </div>
          <h3
            class="css-1puzo0z"
          >
            2024.01.01 ~ 2024.01.31
          </h3>
        </header>
        <div
          class="css-7ttcrf"
        >
          <div
            class="css-135bpbl"
          >
            <header
              class="css-kvjl0w"
            >
              <div
                class="css-vmk80s"
              >
                <svg
                  aria-hidden="true"
                  data-slot="icon"
                  fill="none"
                  stroke="currentColor"
                  stroke-width="1.5"
                  style="width: 0.9rem; height: 0.9rem;"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M3.75 9h16.5m-16.5 6.75h16.5"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                </svg>
                <h4
                  class="css-pqtvky"
                >
                  수입 - 지출
                </h4>
              </div>
              <span>
                7,000,000
              </span>
            </header>
            <section
              class="css-1r2jysi"
            />
          </div>
          <div
            class="css-135bpbl"
          >
            <header
              class="css-1r6e9a4"
            >
              <div
                class="css-vmk80s"
              >
                <svg
                  aria-hidden="true"
                  data-slot="icon"
                  fill="none"
                  stroke="currentColor"
                  stroke-width="1.5"
                  style="width: 0.9rem; height: 0.9rem;"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M12 4.5v15m7.5-7.5h-15"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                </svg>
                <h4
                  class="css-pqtvky"
                >
                  수입
                </h4>
              </div>
              <span>
                10,000,000
              </span>
            </header>
            <section
              class="css-1r2jysi"
            >
              <div
                class="css-uf1ume"
              >
                <span>
                  └ 
                  이달의 수입
                </span>
                <span>
                  8,000,000
                </span>
              </div>
              <div
                class="css-uf1ume"
              >
                <span>
                  └ 
                  전월이월
                </span>
                <span>
                  2,000,000
                </span>
              </div>
            </section>
          </div>
          <div
            class="css-135bpbl"
          >
            <header
              class="css-1i1w77g"
            >
              <div
                class="css-vmk80s"
              >
                <svg
                  aria-hidden="true"
                  data-slot="icon"
                  fill="none"
                  stroke="currentColor"
                  stroke-width="1.5"
                  style="width: 0.9rem; height: 0.9rem;"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M5 12h14"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                </svg>
                <h4
                  class="css-pqtvky"
                >
                  지출
                </h4>
              </div>
              <span>
                3,000,000
              </span>
            </header>
            <section
              class="css-1r2jysi"
            >
              <div
                class="css-uf1ume"
              >
                <span>
                  └ 
                  현금지출
                </span>
                <span>
                  1,000,000
                </span>
              </div>
              <div
                class="css-uf1ume"
              >
                <span>
                  └ 
                  카드지출
                </span>
                <span>
                  2,000,000
                </span>
              </div>
            </section>
          </div>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="css-14chwgd"
    >
      <header
        class="css-kympkc"
      >
        <div
          class="css-359auk"
        >
          <svg
            aria-hidden="true"
            data-slot="icon"
            fill="none"
            stroke="currentColor"
            stroke-width="1.5"
            style="width: 1rem; height: 1rem;"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="m4.5 15.75 7.5-7.5 7.5 7.5"
              stroke-linecap="round"
              stroke-linejoin="round"
            />
          </svg>
          <h2
            class="css-1bt39ed"
          >
            이달의 자산현황
          </h2>
        </div>
      </header>
      <div
        class="css-7ttcrf"
      >
        <div
          class="css-135bpbl"
        >
          <header
            class="css-kvjl0w"
          >
            <div
              class="css-vmk80s"
            >
              <svg
                aria-hidden="true"
                data-slot="icon"
                fill="none"
                stroke="currentColor"
                stroke-width="1.5"
                style="width: 0.9rem; height: 0.9rem;"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M3.75 9h16.5m-16.5 6.75h16.5"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                />
              </svg>
              <h4
                class="css-pqtvky"
              >
                순자산
              </h4>
            </div>
            <span>
              500,000,000
            </span>
          </header>
          <section
            class="css-1r2jysi"
          />
        </div>
        <div
          class="css-135bpbl"
        >
          <header
            class="css-1r6e9a4"
          >
            <div
              class="css-vmk80s"
            >
              <svg
                aria-hidden="true"
                data-slot="icon"
                fill="none"
                stroke="currentColor"
                stroke-width="1.5"
                style="width: 0.9rem; height: 0.9rem;"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M12 4.5v15m7.5-7.5h-15"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                />
              </svg>
              <h4
                class="css-pqtvky"
              >
                자산
              </h4>
            </div>
            <span>
              500,000,000
            </span>
          </header>
          <section
            class="css-1r2jysi"
          >
            <div
              class="css-uf1ume"
            >
              <span>
                └ 
                현금잔액
              </span>
              <span>
                100,000,000
              </span>
            </div>
            <div
              class="css-uf1ume"
            >
              <span>
                └ 
                투자자산
              </span>
              <span>
                350,000,000
              </span>
            </div>
            <div
              class="css-uf1ume"
            >
              <span>
                └ 
                기타자산
              </span>
              <span>
                50,000,000
              </span>
            </div>
          </section>
        </div>
        <div
          class="css-135bpbl"
        >
          <header
            class="css-1i1w77g"
          >
            <div
              class="css-vmk80s"
            >
              <svg
                aria-hidden="true"
                data-slot="icon"
                fill="none"
                stroke="currentColor"
                stroke-width="1.5"
                style="width: 0.9rem; height: 0.9rem;"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 12h14"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                />
              </svg>
              <h4
                class="css-pqtvky"
              >
                부채
              </h4>
            </div>
            <span>
              100,000,000
            </span>
          </header>
          <section
            class="css-1r2jysi"
          >
            <div
              class="css-uf1ume"
            >
              <span>
                └ 
                대출
              </span>
              <span>
                75,000,000
              </span>
            </div>
            <div
              class="css-uf1ume"
            >
              <span>
                └ 
                기타부채
              </span>
              <span>
                25,000,000
              </span>
            </div>
          </section>
        </div>
      </div>
    </section>
    <section
      class="css-14chwgd"
    >
      <header
        class="css-kympkc"
      >
        <div
          class="css-359auk"
        >
          <svg
            aria-hidden="true"
            data-slot="icon"
            fill="none"
            stroke="currentColor"
            stroke-width="1.5"
            style="width: 1rem; height: 1rem;"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="m4.5 15.75 7.5-7.5 7.5 7.5"
              stroke-linecap="round"
              stroke-linejoin="round"
            />
          </svg>
          <h2
            class="css-1bt39ed"
          >
            이달의 가계
          </h2>
        </div>
        <h3
          class="css-1puzo0z"
        >
          2024.01.01 ~ 2024.01.31
        </h3>
      </header>
      <div
        class="css-7ttcrf"
      >
        <div
          class="css-135bpbl"
        >
          <header
            class="css-kvjl0w"
          >
            <div
              class="css-vmk80s"
            >
              <svg
                aria-hidden="true"
                data-slot="icon"
                fill="none"
                stroke="currentColor"
                stroke-width="1.5"
                style="width: 0.9rem; height: 0.9rem;"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M3.75 9h16.5m-16.5 6.75h16.5"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                />
              </svg>
              <h4
                class="css-pqtvky"
              >
                수입 - 지출
              </h4>
            </div>
            <span>
              7,000,000
            </span>
          </header>
          <section
            class="css-1r2jysi"
          />
        </div>
        <div
          class="css-135bpbl"
        >
          <header
            class="css-1r6e9a4"
          >
            <div
              class="css-vmk80s"
            >
              <svg
                aria-hidden="true"
                data-slot="icon"
                fill="none"
                stroke="currentColor"
                stroke-width="1.5"
                style="width: 0.9rem; height: 0.9rem;"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M12 4.5v15m7.5-7.5h-15"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                />
              </svg>
              <h4
                class="css-pqtvky"
              >
                수입
              </h4>
            </div>
            <span>
              10,000,000
            </span>
          </header>
          <section
            class="css-1r2jysi"
          >
            <div
              class="css-uf1ume"
            >
              <span>
                └ 
                이달의 수입
              </span>
              <span>
                8,000,000
              </span>
            </div>
            <div
              class="css-uf1ume"
            >
              <span>
                └ 
                전월이월
              </span>
              <span>
                2,000,000
              </span>
            </div>
          </section>
        </div>
        <div
          class="css-135bpbl"
        >
          <header
            class="css-1i1w77g"
          >
            <div
              class="css-vmk80s"
            >
              <svg
                aria-hidden="true"
                data-slot="icon"
                fill="none"
                stroke="currentColor"
                stroke-width="1.5"
                style="width: 0.9rem; height: 0.9rem;"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 12h14"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                />
              </svg>
              <h4
                class="css-pqtvky"
              >
                지출
              </h4>
            </div>
            <span>
              3,000,000
            </span>
          </header>
          <section
            class="css-1r2jysi"
          >
            <div
              class="css-uf1ume"
            >
              <span>
                └ 
                현금지출
              </span>
              <span>
                1,000,000
              </span>
            </div>
            <div
              class="css-uf1ume"
            >
              <span>
                └ 
                카드지출
              </span>
              <span>
                2,000,000
              </span>
            </div>
          </section>
        </div>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "store": {
    "@@observable": [Function],
    "dispatch": [Function],
    "getState": [Function],
    "replaceReducer": [Function],
    "subscribe": [Function],
  },
  "unmount": [Function],
}
`;
